packages/auth0_api_python/EXAMPLES.md

```diff
@@ -0,0 +1,160 @@
+# Auth0 API Python Examples
+
+This document provides examples for using the `auth0-api-python` package to validate Auth0 tokens in your API.
+
+## Bearer Authentication
+
+Bearer authentication is the standard OAuth 2.0 token authentication method.
+
+### Using verify_access_token
+
+```python
+import asyncio
+from auth0_api_python import ApiClient, ApiClientOptions
+
+async def validate_bearer_token(headers):
+    api_client = ApiClient(ApiClientOptions(
+        domain="your-tenant.auth0.com",
+        audience="https://api.example.com"
+    ))
+    
+    try:
+        # Extract the token from the Authorization header
+        auth_header = headers.get("authorization", "")
+        if not auth_header.startswith("Bearer "):
+            return {"error": "Missing or invalid authorization header"}, 401
+            
+        token = auth_header.split(" ")[1]
+        
+        # Verify the access token
+        claims = await api_client.verify_access_token(token)
+        return {"success": True, "user": claims["sub"]}
+    except Exception as e:
+        return {"error": str(e)}, getattr(e, "get_status_code", lambda: 401)()
+
+# Example usage
+headers = {"authorization": "Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9..."}
+result = asyncio.run(validate_bearer_token(headers))
+```
+
+### Using verify_request
+
+```python
+import asyncio
+from auth0_api_python import ApiClient, ApiClientOptions
+from auth0_api_python.errors import BaseAuthError
+
+async def validate_request(headers):
+    api_client = ApiClient(ApiClientOptions(
+        domain="your-tenant.auth0.com",
+        audience="https://api.example.com"
+    ))
+    
+    try:
+        # Verify the request with Bearer token
+        claims = await api_client.verify_request(
+            headers=headers
+        )
+        return {"success": True, "user": claims["sub"]}
+    except BaseAuthError as e:
+        return {"error": str(e)}, e.get_status_code(), e.get_headers()
+
+# Example usage
+headers = {"authorization": "Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9..."}
+result = asyncio.run(validate_request(headers))
+```
+
+
+## DPoP Authentication 
+
+**DPoP (Demonstrating Proof-of-Possession)** is a security extension that binds access tokens to cryptographic keys, preventing token theft and replay attacks.
```

Comment:
```suggestion
**DPoP (Demonstrating Proof-of-Possession)** is a security extension that binds access tokens to cryptographic keys, preventing the replay of stolen or leaked access tokens.
```

Reply:
It doesn't prevent token theft.

Reply:
That means it protects against the consequences of token theft (using them to access protected resources), but it doesn't prevent the token theft itself.